### ISSUE TITLE
Include libunity in Linux target

### ISSUE THREAD

**Issue by JakobDev** [electron-userland/electron-builder#8019]
* **Electron-Builder Version**: 
24.9.1

* **Node Version**: 
21.5.0

* **Electron Version**:
28.9.0

* **Electron Type (current, beta, nightly)**:
stable

* **Target**: 
Linux

Electron [needs libunity](https://github.com/electron/electron/blob/fc917985ae8165e0cbcc11b3bebbcc995b1bd271/shell/browser/linux/unity_service.cc#L70) for showing a badge and the progress in the window icon. This lib should be included in the Linux target, as it is not pre-installed on most systems.

**Comment by mmaietta** [comment]
This sounds like a new requirement in newer electron versions? Can you try adding `libunity` to the linux `depends` configuration and see if that resolves your issue? Would like confirmation first before making changes to the code base.

**Comment by JakobDev** [comment]
> This sounds like a new requirement in newer electron versions? 

No. This is very old, but it is not documented. It is needed for [setProgressBar](https://www.electronjs.org/de/docs/latest/api/browser-window#winsetprogressbarprogress-options) and [setBadgeCount](https://www.electronjs.org/de/docs/latest/api/browser-window#winsetprogressbarprogress-options) to work on Linux. If Electron can't find this lib, these function just don't work and Electron don't show a Error at all.

> Can you try adding `libunity` to the `linux` depends configuration and see if that resolves your issue?

You mean the DEB package?

**Comment by JakobDev** [comment]
Push

**Comment by mmaietta** [comment]
@JakobDev I found where the default values are coming from, they were located in an upstream dependency app-builder-bin. I've moved the logic into electron-builder, just need to know from you where `libunity` is supposed to go. Which target does it apply to?
https://github.com/electron-userland/electron-builder/blob/a86afcde04c8f837bd93b88238f01e9bf35e825d/packages/app-builder-lib/src/targets/FpmTarget.ts#L288-L311
